model-optimizer/extensions/ops/RandomUniformOp.py

```diff
@@ -13,14 +13,21 @@
  See the License for the specific language governing permissions and
  limitations under the License.
 """
+from mo.graph.graph import Graph
+from mo.ops.op import Op
+from mo.front.common.partial_infer.random_uniform import tf_random_uniform_infer
```

Comment:
Remove file `mo/front/common/partial_infer/random_uniform.py` and use `copy_shape_infer` method.